### PR TITLE
Write Django cmd for running lint

### LINKED ISSUE #82: Address local runs of tests in `dev`
We need a way for people to run the things that run in CI locally so they don't have to go through CI over and over. Options: 

- A template pre-commit hook that runs the same things as in CI (would have to be added individually) 
- Use something like [`act`](https://github.com/nektos/act) to run the github actions locally
- A script that is used locally and in CI
- A requirements-dev.txt file and freeze may be necessary in all cases to pin requirements in dev and CI.

### PR DESCRIPTION
**Pull request by SSPJ**
## 🗣 Description ##

A script to allow running linters on dev. And documentation.

## 💭 Motivation and context ##

Devs need to be able to check their work before pushing it to GitHub.

Closes #82

### CODE REVIEW

File: docs/developer/README.md [hunk 1]
```diff
@@ -34,4 +34,20 @@ You can change the logging verbosity, if needed. Do a web search for "django log
 
 ## Running tests
 
-tbd.
+Django's test suite:
```
Comment (loganmeetsworld) [inline docs/developer/README.md:37]: Add a note to make sure Docker is running and that this has to be run from `src`?

File: docs/developer/README.md [hunk 2]
```diff
@@ -34,4 +34,20 @@ You can change the logging verbosity, if needed. Do a web search for "django log
...
+docker-compose up -d
+docker-compose exec app ./manage.py test
+docker-compose stop
+```
+
+Linters:
+
+```shell
+docker-compose up -d
+docker-compose exec app ./manage.py test
```
Comment (loganmeetsworld) [inline docs/developer/README.md:49]: Should this be `lint`?

**Review by loganmeetsworld — APPROVED** [review]
Works for me! A couple notes.

File: docs/developer/README.md (hunk 1 above)
Comment (SSPJ) [reply docs/developer/README.md]: Added!

File: docs/developer/README.md (hunk 2 above)
Comment (SSPJ) [reply docs/developer/README.md:50]: Yes! Vielen Dank!

**Pull request merged**